src/ops.rs

```diff
@@ -23,6 +23,33 @@ pub fn check_command_exists(command: &str) -> Result<()> {
     }
 }
 
+pub fn check_file_modified(file: &str) -> Result<bool> {
```

Comment:
I'm not sure if checking for a modification known to git in the Containerfile (if I'm interpreting this code correctly) is a good way to determine whether the file is safe to overwrite. In the case that a user is migrating from the legacy template, they'll probably have all changes commited to git..

I'd rather check only if the file is in `.gitignore` and if it has the bluebuild label.

- File in `.gitignorr`, has label -> build normally
- File in `.gitignore`, no label -> build normally
- File not in `.gitignore`, has label -> "It looks you have a BlueBuild-generated Containerfile that is not .gitignored. Do you want to remove it from git and add it to .gitignore (Y, n)?" build either way
- File not in `.gitignore`, doesn't have label -> *error message one similar to the one in my earlier message* only build on yes

Reply:
Great! Looking sweet!